### PR TITLE
docs: (core/i18n): [translate] Done

### PR DESCRIPTION
**Pull request by DarrenWong**
##### Checklist

- [x] documentation is changed or added

##### Affected core subsystem(s)
docs/source/en/core/i18n.md

##### Description of change
Translated https://github.com/eggjs/egg/blob/master/docs/source/zh-cn/core/i18n.md, thanks for any comments and reviews

### CODE REVIEW

File: docs/source/en/core/i18n.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+exports.i18n = {
+  defaultLocale: 'zh-CN',
+};
+```
+
+## Switch Language
+
+Here we have some ways to switch the application's current language (Modified records will set to the cookie `locale`), can be used directly in the next request.
+
+Higher Priority:
```
Comment (iyuq) [inline docs/source/en/core/i18n.md]: Priority from high to low?

File: docs/source/en/core/i18n.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+```js
+// config/config.default.js
+exports.i18n = {
+  defaultLocale: 'zh-CN',
+};
+```
+
+## Switch Language
+
+Here we have some ways to switch the application's current language (Modified records will set to the cookie `locale`), can be used directly in the next request.
```
Comment (iyuq) [inline docs/source/en/core/i18n.md]: the next request will use the language been set directly

Comment (DarrenWong) [inline docs/source/en/core/i18n.md]: oh thx, i might mistake the meaning. how about this?
the next request will use the language setting directly

File: docs/source/en/core/i18n.md (hunk 1 above)
Comment (DarrenWong) [inline docs/source/en/core/i18n.md]: Yes from the chinese translate is ok.
But I also think both okay, if refer use `Priority from high to low`, I can modify

File: docs/source/en/core/i18n.md (hunk 2 above)
Comment (iyuq) [inline docs/source/en/core/i18n.md]: LGTM

**Pull request merged**